discovery-provider/src/api/v1/tracks.py

```diff
@@ -430,9 +430,7 @@ def get(self):
             "only_downloadable": args["only_downloadable"],
         }
         response = search(search_args)
-        tracks = response["tracks"]
-        tracks = list(map(extend_track, tracks))
-        return success_response(tracks)
+        return success_response(response)
```

Comment:
Are we sure we can drop all these calls to `extend_search` functions?  I suspect we should keep them as is.  What's the reason to remove them?

Reply:
Just adding it to search() so we don't have to call them together every time.